app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Hi @AbuRayhaan, based on the requirements of this project, the application should request the user to confirm the password. To achieve this, `confirmable` should be enabled in the user model that's

```.ruby
 devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :confirmable
```